core/chain-configs/src/genesis_validate.rs

```diff
@@ -152,12 +152,12 @@ mod test {
     fn test_total_supply_not_match() {
         let mut genesis = Genesis::default();
         genesis.config.validators = vec![AccountInfo {
-            account_id: "test".parse().unwrap(),
+            account_id: AccountId::test_account(),
```

Comment:
tbh, this change feels like a regression to me:

* more chars to type
* now the actual name of the account is obscured
* AccountId now concernts itself with the stuff which is only relevant to tests in unrelated crates.

Reply:
Valid points, I'll go ahead to drop all the static_account methods all-together.